runtime/gc_ctrl.c

```diff
@@ -643,18 +640,20 @@ void caml_init_gc (uintnat minor_size, uintnat major_size,
                    uintnat custom_maj, uintnat custom_min,
                    uintnat custom_bsz)
 {
-  uintnat major_heap_size =
-    Bsize_wsize (caml_normalize_heap_increment (major_size));
+  uintnat major_bsize;
+  if(major_size < Heap_chunk_min) major_size = Heap_chunk_min;
```

Comment:
OK to inline `caml_normalize_heap_increment` while fixing the way it's used.  However, the house style is `if (` with a space between `if` and the condition.